index.html

```diff
@@ -356,11 +363,14 @@
 
     let sup_districts = {};
     let tally_types = {};
+    const precinct_to_sup_district_map = precincts_to_sup_districts(election.fields);
     for (let ballot of election.ballots) {
         if (ballot.contest != contest) {
             continue;
         }
-        let district = precinct_to_sup_district(ballot.precinct, election.fields);
+        let district = precinct_to_sup_district_map.get(ballot.precinct);
+        if (district == null)
```

Comment:
I've tried to stick to consistently always bracing if-bodies.

Reply:
ok